src/sass/app/components/peek_box.scss

```diff
@@ -0,0 +1,306 @@
+$PEEKBOX_GRID_ITEMS: mask, topright, topleft, top, left, right, bottom, bottomleft, bottomright;
+/* stylelint-disable color-no-hex */
```

Comment:
Maybe the put colors in constants? And  `/* stylelint-disable color-no-hex */` should be re-enabled after this block.

Reply:
Since they are not going to be used anywhere else other than the peek_box, I believe this a better way to put everything beside each other. Enabling stylelint after that tho :)